packages/permission-controller/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Changed
+- Restrict certain permissions by subject type ([#1233](https://github.com/MetaMask/core/pull/1233))
```

Comment:
Nit: The subject type restriction sounds more like a feature to me. It has zero impact on existing usages, but it allows for this new type of restriction

```suggestion
### Added
- Allow restricting permissions by subject type ([#1233](https://github.com/MetaMask/core/pull/1233))

### Changed
```